observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -5,3 +5,143 @@ include::_attributes/common-attributes.adoc[]
 :context: logging-6x
 
 toc::[]
+
+The `ClusterLogForwarder` (CLF) allows users to configure forwarding of logs to various destinations. It provides a flexible way to select log messages from different sources, send them through a pipeline that can transform or filter them, and forward them to one or more outputs.
+
+.Key Functions of the ClusterLogForwarder
+* Selects log messages using inputs
+* Forwards logs to external destinations using outputs
+* Filters, transforms, and drops log messages using filters
+* Defines log forwarding pipelines connecting inputs, filters and outputs
+* Provides global configuration options
+
+== Structuring the ClusterLogForwarder
+
+The CLF has a `spec` section that contains the following key components:
+
+Inputs:: Select log messages to be forwarded. Built-in input types `application`, `infrastructure` and `audit` forward logs from different parts of the cluster. You can also define custom inputs.
+
+Outputs:: Define destinations to forward logs to. Each output has a unique name and type-specific configuration.
+
+Pipelines:: Define the path logs take from inputs, through filters, to outputs. Pipelines have a unique name and consist of a list of input, output and filter names.
+
+Filters:: Transform or drop log messages in the pipeline. Users can define filters that match certain log fields and drop or modify the messages. Filters are applied in the order specified in the pipeline.
+
+== Configuring Inputs
+
+Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
+
+application:: Selects logs from all application containers, except those in infrastructure namespaces like `openshift-*` and `kube-*`.
+
+infrastructure:: Selects logs from infrastructure components running in `openshift-*` and `kube-*` namespaces and node logs.
+
+audit:: Selects logs from the OpenShift API server audit logs, Kubernetes API server audit logs and node audit logs from auditd.
```

Comment:
Selects logs from the OpenShift API server audit logs, Kubernetes API server audit logs, ovn audit logs and node audit logs from auditd.